x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/list_common/exception_item_card/meta/meta.tsx

```diff
@@ -0,0 +1,161 @@
+          data-test-subj={`${dataTestSubj}-actionItem-${reference.id}`}
+          key={reference.id}
+        >
+          <EuiToolTip content={reference.name} anchorClassName="eui-textTruncate">
+            <SecuritySolutionLinkAnchor
+              data-test-subj="ruleName"
+              deepLinkId={SecurityPageName.rules}
+              path={getRuleDetailsTabUrl(reference.id, RuleDetailTabs.alerts)}
+            >
+              {reference.name}
+            </SecuritySolutionLinkAnchor>
+          </EuiToolTip>
+        </EuiContextMenuItem>
+      ));
+    }, [references, dataTestSubj]);
+
+    return (
+      <EuiFlexGroup
+        alignItems="center"
+        responsive={false}
+        gutterSize="s"
+        data-test-subj={dataTestSubj}
+      >
+        <StyledFlexItem grow={false}>
+          <MetaInfoDetails
+            fieldName="created_by"
+            label={i18n.EXCEPTION_ITEM_CARD_CREATED_LABEL}
+            value1={<FormattedDate fieldName="created_at" value={item.created_at} />}
+            value2={item.created_by}
+            dataTestSubj={`${dataTestSubj}-createdBy`}
+          />
+        </StyledFlexItem>
+        <StyledFlexItem grow={false}>
+          <MetaInfoDetails
+            fieldName="updated_by"
+            label={i18n.EXCEPTION_ITEM_CARD_UPDATED_LABEL}
+            value1={<FormattedDate fieldName="updated_at" value={item.updated_at} />}
+            value2={item.updated_by}
+            dataTestSubj={`${dataTestSubj}-updatedBy`}
+          />
+        </StyledFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiPopover
+            button={
+              <EuiButtonEmpty
+                onClick={onAffectedRulesClick}
+                iconType="list"
+                data-test-subj={`${dataTestSubj}-affectedRulesButton`}
+              >
+                {i18n.AFFECTED_RULES(references.length)}
+              </EuiButtonEmpty>
+            }
+            panelPaddingSize="none"
+            isOpen={isPopoverOpen}
+            closePopover={onClosePopover}
+            data-test-subj={`${dataTestSubj}-items`}
+          >
+            <EuiContextMenuPanel size="s" items={itemActions} />
+          </EuiPopover>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    );
+  }
+);
+ExceptionItemCardMetaInfo.displayName = 'ExceptionItemCardMetaInfo';
+
+interface MetaInfoDetailsProps {
+  fieldName: string;
+  label: string;
+  value1: JSX.Element | string;
+  value2: string;
+  dataTestSubj: string;
+}
+
+const MetaInfoDetails = memo<MetaInfoDetailsProps>(({ label, value1, value2, dataTestSubj }) => {
+  return (
+    <EuiFlexGroup alignItems="center" gutterSize="s" wrap={false} responsive={false}>
+      <EuiFlexItem grow={false}>
+        <EuiText size="xs" style={{ fontFamily: 'Inter' }}>
+          {label}
+        </EuiText>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false} data-test-subj={`${dataTestSubj}-value1`}>
+        <EuiBadge color="default" style={{ fontFamily: 'Inter' }}>
```

Comment:
As far as I see in the [EUI Guide](https://elastic.github.io/eui/#/theming/typography/values) it's recommended to use euiTheme for the font famility setting.